src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1284,18 +1284,25 @@ protected override void Validate(object arguments, EngineIntrinsics engineIntrin
                     null, Metadata.ValidateCountNotInArray);
             }
 
+            if (MinLength == MaxLength && len != MaxLength)
```

Comment:
Thanks! If @daxian-dbw  and @mklement0 do not object I'll create new PR tomorrow to only remove unneeded resource strings.